src/astProcessor.ts

```diff
@@ -15,11 +15,48 @@ export class AstProcessor {
     }
   }
 
+  private getFeatureFromDoc(doc: any) : Feature {
+    const feature = {
+      name: doc.feature.name,
+      position: doc.feature.location.line,
+      uiElements: this.getUIElementsFromFeature(doc.feature.uiElements)
+    }
+
+    return feature
+  }
+
+  private getUIElementsFromFeature(uiElements: Array<any>) : Array<Element> {
```

Comment:
- [ ] This method name does not appear to do what its name tells. I suggest renaming it.
- [ ] Can the array `uiElements` be typed? BTW, it's simpler to use the `[]` notation for arrays, instead of `Array<>`.

Reply:
I think it will be hard to type `uiElements`. Maybe, in this case, would be easier import the interface from `concordialang-types`. I had the same dificulty with `intefaces/document.ts` (next commits).

Reply:
ok